src/main/java/io/github/moulberry/notenoughupdates/commands/dungeon/MapCommand.java

```diff
@@ -50,6 +50,12 @@ public MapCommand() {
 
 	@Override
 	public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+		if (!NotEnoughUpdates.INSTANCE.config.hidden.dev) {
```

Comment:
if there is an exception for dev mode here shouldn't there also be an exception for it when checking to override the config?